data/gaTable.yaml

```diff
@@ -567,7 +567,7 @@
   analyticsContact: 'David Watkins'
   analyticsContactEmail: 'wwatkins@usgs.gov'
 
-- longName: Grand Canyon Monitoring Research Center Website
+- longName: Grand Canyon Monitoring and Research Center''s Discharge, Sediment, and Water Quality Monitoring App
```

Comment:
I see another typo I made, if you can confirm I'll put another PR up without that. Thanks!